libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -1206,10 +1206,11 @@ uint16_t usType = 0U;
 		if( ( pxDNSMessageHeader->usFlags & dnsRX_FLAGS_MASK ) == dnsEXPECTED_RX_FLAGS )
 		{
 			const uint16_t usCount = ( uint16_t ) ipconfigDNS_CACHE_ADDRESSES_PER_ENTRY;
+			uint16_t usNumARecordsStored = 0;
 
-			for( x = 0U; ( x < pxDNSMessageHeader->usAnswers ) && ( x < usCount ); x++ )
+			for( x = 0U; ( x < pxDNSMessageHeader->usAnswers ) && ( usNumARecordsStored < usCount ); x++ )
 			{
-			BaseType_t xDoAccept;
+                BaseType_t xDoAccept;
 
 				uxResult = prvSkipNameField( pucByte,
 											 uxSourceBytesRemaining );
```

Comment:
White spacing seems to be off for the xDoAccept variable and the uxResult assignment.

Reply:
This file uses tabs unfortunately.  I have updated my changes to use tabs rather than spaces.